linux_os/guide/system/software/integrity/software-integrity/aide/aide_verify_ext_attributes/rule.yml

```diff
@@ -53,3 +53,10 @@ ocil: |-
     To determine that AIDE is verifying extended file attributes, run the following command:
     <pre>$ grep xattrs /etc/aide.conf</pre>
```

Comment:
This wouldn't show all rules lines, but only the compliant lines. It should be also updated to use a regular expression similar to the one used in the OVAL check.